python-backend/app/api/mines/namespace/mines.py

```diff
@@ -21,7 +22,10 @@
 
 api = Namespace('mines', description='Mine related operations')
 
-api.add_resource(MineResource, '', '/<string:mine_no_or_guid>')
+api.add_resource(MineResource, '/<string:mine_no_or_guid>')
+api.add_resource(MineListResource, '')
+api.add_resource(MineMapResource, '/map-list')
```

Comment:
I don't think this is an appropriate endpoint for our API to have. The API knows nothing about the map on the frontend. It's probably more reasonable for this to be something like `(MineBasicInfoResource, '/basic')`

Not sure if we're already referring to something as `BasicInfo` elsewhere. I'm not committed to that name if we are -- but I think it serves the purpose of illustrating my point

Reply:
mines/coordinates?

Reply:
I'd like to see this changed, but I might just split it into a separate tech debt ticket that deals with removing references to the map from the backend since those references exist in a few places